docs/changelog.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changelog
 - Docker support
 
+### Changed
+
+- Migrate to API of the OSTIS Platform 0.10.0
```

Comment:
shouldn't at least one mention of ostis-ui-lib be updated? and shouldn't this change be mentioned here?